.github/workflows/build-updatesite-master.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow builds the plugin, feature and update site and pushes the update
+# site to a dedicated branch called "updatesite" on the Githu repo.
```

Comment:
It's good to merge :) The only typo I found is "Githu" repo, at the end of this line (sorry about the annoyance) :P

Reply:
damn. Thank you for finding it! :)

Reply:
Should've been fixed in both workflow files. :)